redis/db/redisbp/hooks_test.go

```diff
@@ -1,27 +1,34 @@
-package redisbp_test
+package redisbp
 
 import (
 	"context"
 	"testing"
 
 	"github.com/go-redis/redis/v8"
 	"github.com/opentracing/opentracing-go"
+	"github.com/prometheus/client_golang/prometheus"
 
-	"github.com/reddit/baseplate.go/redis/db/redisbp"
+	"github.com/reddit/baseplate.go/prometheusbp/promtest"
 	"github.com/reddit/baseplate.go/thriftbp"
 	"github.com/reddit/baseplate.go/tracing"
 )
 
 func TestSpanHook(t *testing.T) {
 	ctx, _ := thriftbp.StartSpanFromThriftContext(context.Background(), "foo")
-	hooks := redisbp.SpanHook{ClientName: "redis"}
+	hooks := SpanHook{ClientName: "redis"}
 	statusCmd := redis.NewStatusCmd(ctx, "ping")
 	stringCmd := redis.NewStringCmd(ctx, "get", "1")
-	stringCmd.SetErr(redis.Nil)
+	stringCmd.SetErr(nil)
```

Comment:
@pacejackson I changed this because the [doc](https://pkg.go.dev/github.com/go-redis/redis/v8#Nil) says `redis.Nil` is what redis returns when the key does not exist, which makes me think it's an error but it's actually not and the error we get in the hook is nil, which is kinda confusing so I think just use `nil` here is more clear.